entity_event_generator.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  entity_event_lambda_name = "${local.environment}-entity-event-generator-lambda"
+  entity_event_topic_name  = "${local.environment}-entity-event-generator-topic"
```

Comment:
Would this be better named without `generator`? In our Alpha we used `entity-event-topic`.